lib/plugins/provisioning/src/index.js

```diff
@@ -43,6 +42,9 @@ const rtype = function *(rid) {
   return rid;
 };
 
+const wscope = { system: ['abacus.usage.write'] };
```

Comment:
We should start naming variables with longer and more descriptive names. What about `writeScope` and `readScope`?